src/sink/kafka.rs

```diff
@@ -131,14 +132,15 @@ impl KafkaPublishable for KafkaPublishResult {
     }
 }
 
-type BoxedKafkaPublishable = Box<KafkaPublishable + Send + Sync>;
```

Comment:
Why is it safe to get rid of Sync here?

Reply:
`Sync` allows types to be shared my multiple threads, `Send` means it can be sent between threads. In our case we don't need to use it again after sending to a different thread so `Send` is fine. Otherwise we would need to make the rdkafka `OwnedHeaders` `Sync` somehow and I didn't know how to do that so this seemed the easiest solution.